report.py

```diff
@@ -529,6 +529,9 @@ def order_dict(d):
         Given a dict whose values are either (messages, words) or just an int
         turn it into an ordered dict ordered from key with most to least
         """
+        if type(d) != dict:
+            print("Type of {} is not a dict!".format(d))
```

Comment:
In what case is `d` not of type `dict`?

Reply:
Ugh, this is annoying.  

In create_key(), we do
                    cur[k] = collections.defaultdict(int)
And that turns out to be correct *most* of the time.  But sometimes, actually, it should be a dict :)

Reply:
So if I'm understanding correctly, this function is turning `dict` into `OrderedDict`, but if the parameter is a `defaultdict`, then we return empty `dict`. Is that right?

Reply:
That's correct, @jdq22 

There's a design principle that I adhered to that may not be right, which is basically "by default do the right thing," rather than "enforce doing the right thing."  Or, said differently, "magic."  This is a kludgefix for that magic.

Reply:
I think I understand based on where I'm seeing this function called. Will you remove the debug statement about the type of `d`? And I think we want in the docblock an explanation that we can pop out of this function IFF the parameter is not a `dict`.

Reply:
Done.